guides/development-workflow.md

```diff
@@ -0,0 +1,341 @@
+- spending too much time over engineering (gold plating) our code
+- delivering too quickly at the cost of any code quality
+
+#### The twenty-minute rules
+
+  1. being helpful is one of our values and this extends to development. If after spending 20 minutes on a problem you haven't made progress, invoke the twenty-minute rule with another developer and get their advice
+
+  2. discussions that go on for twenty-minutes or more can be a time sink. If an agreement on how to progress cannot be found after 20 minutes, mentioning this rule provides a pre-agreed way to acknowledge the problem and get to a resolution as quickly as possible. Rate your concern for the issue out of 10 and a next step should naturally emerge
+
+---
+
+## The lifecycle of a story
+
+### Start work on a new story
+
+1. assign yourself to the Trello card
+2. move the card to Trello's 'in progress' column
+3. [make a new branch](#how-to-make-a-branch) off of `develop`
+
+  ```
+  git checkout -b <branch-name> origin/develop
+  ```
+
+4. meet the acceptance criteria
+5. [commit](#committing) your code
+
+### Submit your story for code review
+
+1. push your branch to the remote repository
+
+  ```
+  git push origin <branch-name>
+  ```
+
+2. create a [pull request](##pull-requests) into the `develop` branch:
+  * [GitHub](https://docs.gitlab.com/ee/gitlab-basics/add-merge-request.html)
+  * [GitLab](https://docs.gitlab.com/ee/gitlab-basics/add-merge-request.html)
+3. Link the Trello card to the new pull request, linking directly or by using the Trello [power up](http://help.trello.com/article/1021-integrating-trello-with-zendesk)
+3. move the Trello card into the 'Awaiting review column'
+
+### Reviewing another developer's pull request
+
+1. move the Trello card into the 'Under code review' column
+2. find and read the Trello card's acceptance criteria
+3. find and read through the pull request, carrying out a [code review](#code-review)
+  * if it is ready:
+    - press the merge button
+    - delete the leftover branch
+    - move the Trello card into 'Deployed to Staging'
+  * if it's not ready:
+    - leave feedback
+    - mention the author so they're aware
+
+---
+## Working on a ticket
+
+Tickets can fall into two types of work flow depending on their severity.
+
+Most commonly tickets will simply fall into the same [story process](#the-lifecycle-of-a-story), where fixes will go through `develop` and Staging for review and acceptance.
+
+They can however be sometimes when there's an emergency which requires a change to production ASAP. This is when we reach for a [hotfix](#hotfix).
+
+### Hotfix
+
+We think of hotfixes in the same way as [Wikipedia](https://en.wikipedia.org/wiki/Hotfix) describes them:
+
+> a hotfix implies that the change may have been made quickly and outside normal development and testing processes
+
+> A hotfix is a change deemed critical enough that it cannot be held off until a regular content patch.
+
+An emergency for would be when:
```

Comment:
This sentence doesn't make sense to me. Should it be `An emergency for instance would be when`?